tests/test_unclassified_job.py

```diff
@@ -26,3 +25,13 @@ def test_open_unclassified_failure_log(base_url, selenium):
     treeherder_page.open_next_unclassified_failure()
     logviewer_page = treeherder_page.job_details.open_logviewer()
     assert logviewer_page.is_job_status_visible
+
+
+@pytest.mark.nondestructive
+def test_view_unclassified_jobs(base_url, selenium):
+    page = TreeherderPage(selenium, base_url).open()
+    assert len(page.result_sets[0].testfailed_job_title) >= 0
```

Comment:
The test is to check unclassified failures, and yes it will fail if there aren't any test failures. As with a number of other Treeherder tests I'm not entirely sure how to gracefully fail if the data I'm looking for doesn't happen to exist. 
I'm unclear what you mean about using the jobs property in this way. The filter button returns all types of unclassified failures. I guess to be thorough I should return to verifying that all types of unclassified failures display after the filter, but again there's the test data problem.